src/processing.py

```diff
@@ -0,0 +1,18 @@
+from typing import Dict, List
+
+
+def filter_by_state(list_dict: list, value_key: str = "EXECUTED") -> List[Dict]:
```

Comment:
Функция принимает не просто список, а список словарей
```suggestion
def filter_by_state(list_dict: list[dict], value_key: str = "EXECUTED") -> list[dict]:
```
Ниже то же